Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.view.html

```diff
@@ -99,28 +99,50 @@ <h4 id="systemUsageHeaderName">{{systemUsageName}}</h4>
                            data-field="expirationDate">
                 </div>
                 <div class="form-group">
-                    
-                    <select-simple data-id="lifeCycleStatus"
-                                   data-label="Status"
-                                   data-options="lifeCycleStatusOptions"
-                                   data-ng-model="usageViewModel.lifeCycleStatus"
-                                   data-ng-disabled="!hasWriteAccess"
-                                   data-auto-save-url="{{::autoSaveUrl}}"
-                                   data-field="lifeCycleStatus">
-                    </select-simple>
+                    <label for="lifeCycleStatus" class="control-label small">Livscyklus</label>
+                    <single-select-select2 id="lifeCycleStatus" ng-model="lifeCycleStatusModel" />
+                </div>
+                <div ng-if="validationStatus">
+                    <label class="control-label small">Aktivt</label>
+                    <!-- Show the simple version of the status if there are no details to explain-->
+                    <div class="form-group object-status-active-summary" ng-if="validationStatus.valid && validationStatus.errors.length === 0">
+                        <!--TODO: Change text to danish-->
```

Comment:
Take a good look at this block and compare it to the source of your copy paste.. I hope you will be able to spot that you added complexity that is not part of the it-system usage's validation model... remember it-systems are either valid or invalid... no valid due to errors...

This is what I meant the last time I spoke of copy paste.. You must still understand the code you are taking.